typescript/ecs/cluster/index.ts

```diff
@@ -0,0 +1,24 @@
+import ec2 = require('@aws-cdk/aws-ec2');
```

Comment:
This example is not linked from the readme?

Also, I'd rather have the examples scenario-based, and "cluster" isn't really a use case  someone has.

Isn't starting a cluster subsumed by other examples as well?

Reply:
Ah, right. I assume we should add a link on readme for all examples then?

Cluster is subsumed by other examples, but I thought there was no harm in showing a very basic example given that cluster may be something imported/exported or provisioned separately from services.

Reply:
I changed it so it's not just adding the default ASG capacity, which gives this example a  bit more value since the other examples just use the default capacity.